cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-scaling.adoc

```diff
@@ -0,0 +1,259 @@
+= Tutorial: Scaling an application
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-scaling
+:source-highlighter: pygments
+:pygments-style: emacs
+:icons: font
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2024-04-10
+
+== Scaling
+
+You can manually scale your pods or you can automatically scale your pods by using the Horizontal Pod Autoscaler (HPA). You can also scale your cluster nodes.
+
+=== Manual pod scaling
+
+You can manually scale the pods and nodes of your applications:
+
+* by changing your replicaset and deployment definition 
+* by using the command line
+* by using the web console
+* by changing your *deployment* definition in the `ostoy-frontend-deployment.yaml`
+
+This workshop starts by using only one pod for the microservice. By using one pod, the Kubernetes Replication Controller strives to keep one pod alive. You can also define pod autoscaling by using the link:https://docs.openshift.com/container-platform/latest/nodes/pods/nodes-pods-autoscaling.html[Horizontal Pod Autoscaler](HPA). The HPA is based on workload, and it expands past defined limits if needed.
+
+.Prerequisites
+
+* Create a cluster with the xref:../../cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-lab-overview.adoc[OSToy microservice deployed].
+* Download the link:https://www.rosaworkshop.io/ostoy/yaml/ostoy-microservice-deployment.yaml[ostoy-microservice-deployment.yaml].
+
+.Procedure 
+
+. In the OSToy app, click the *Networking* tab in the left menu.
+. In the "Intra-cluster Communication" section, there is a box that randomly changes colors. Inside the box, you see the microservice's pod name. There is only one box shown because there is only one microservice pod. 
++
+image::deploy-scale-network.png[HPA Menu]
++ 
+Now, scale up the microservice pods and observe the number of boxes change.
+
+. Run the following command to confirm that there is only one pod running on the microservice:
++
+[source,terminal,highlight='4']
+----
+$ oc get pods
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                  READY     STATUS    RESTARTS   AGE
+ostoy-frontend-679cb85695-5cn7x       1/1       Running   0          1h
+ostoy-microservice-86b4c6f559-p594d   1/1       Running   0          1h
+----
+
+. Now, scale the application by using the CLI or using the Web UI:
++
+** Run the following command increases the number of pods from `1` to `3`:
++
+[source,terminal]
+----
+$ oc scale deployment ostoy-microservice --replicas=3
+----
++
+** In the OSToy application, from the left menu, click *Workloads > Deployments > ostoy-microservice*.  On the left side of the page, you see a blue circle with a "1 Pod" label in the middle. Selecting the arrows next to this scales the number of pods. Select the up arrow to `3`.
++
+image::deploy-scale-uiscale.png[UI Scale]
```

Comment:
Depending on what you want the flow to be this screenshot might need updating.  Are there now 4 pods, or 2?  Also, it might be helpful to the reader to pinpoint where they should be looking to see the correct number as there are 2 lines in this screenshot.